driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/tgkill.bpf.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(tgkill_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, TGKILL_E_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_TGKILL_E, TGKILL_E_SIZE);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: pid (type: PT_PID) */
+	pid_t pid = (s32)extract__syscall_argument(regs, 0);
```

Comment:
minor: instead `pid`, `tgid` could be used for consistency

Reply:
ty i will change it!

Reply:
do you like to change the variable name as well?

Reply:
I've done it! thank you!

Reply:
Thanks :-)